### ISSUE TITLE
twilio@4.0.1: decompress error

### ISSUE THREAD

**Issue by progolu** [twilio/scoop-twilio-cli#15]
i tried  scoop install twilio from my powershell and this error comes out 
Please try again or create a new issue by using the following link and paste your console output:
https://github.com/twilio/scoop-twilio-cli/issues/new?title=twilio%404.0.1%3a+decompress+error. dont know what to do next

**Comment by shrutiburman** [comment]
Hi @progolu, Thanks for reaching out. We've identified this issue internally and are working on it. Meanwhile, to unblock you can download the updated cli version from releases executables present [here](https://github.com/twilio/twilio-cli/releases/tag/4.0.1#:~:text=12%20days%20ago-,twilio%2D4.0.1.exe,-46.3%20MB).

**Comment by shrutiburman** [comment]
Hi @progolu ,
This has been fixed in our latest release 🚀.
Closing this issue, feel free to re-open if you're still facing this. 
![Screenshot 2022-08-25 at 12 16 06 PM](https://user-images.githubusercontent.com/87537688/186598439-88aa8cdc-ae47-4326-a947-eb9f823aa2ae.png)
